README.rst

```diff
@@ -35,32 +44,54 @@ Next, configure your aws credentials and default region:
     Default region name [us-east-1]:
     Default output format [None]:
 
-Then, run pcluster configure:
+Then, run ``pcluster configure``. A list of valid options will be displayed for each
+configuration parameter: type an option number and press ``Enter`` to confirm your choice, or just press ``Enter``
+to accept the default option.
 
 .. code-block:: ini
 
   $ pcluster configure
-  Cluster Template [default]:
-  Acceptable Values for AWS Region ID:
-      ap-south-1
-      ...
-      us-west-2
+  INFO: Configuration file /dir/conf_file will be written.
+  Press CTRL-C to interrupt the procedure.
+
+
+  Allowed values for AWS Region ID:
+  1. eu-north-1
+  ...
+  15. us-west-1
+  16. us-west-2
   AWS Region ID [us-east-1]:
-  VPC Name [myvpc]:
-  Acceptable Values for Key Name:
-    keypair1
-    keypair-test
-    production-key
-  Key Name []:
-  Acceptable Values for VPC ID:
-    vpc-1kd24879
-    vpc-blk4982d
-  VPC ID []:
-  Acceptable Values for Master Subnet ID:
-    subnet-9k284a6f
-    subnet-1k01g357
-    subnet-b921nv04
-  Master Subnet ID []:
+  ...
+
+Make sure to select a region on which you own at least an EC2 key pair.
+
+During the process you will be asked to setup your networking environment. The wizard will
+offer you the possibility of creating a new VPC on the fly.
+
+.. code-block:: ini
+
+  Automate VPC creation? (y/n) [n]:
+
+Enter '``n``' if you already have a correctly set up VPC. Otherwise you can let ``pcluster configure``
+create a VPC for you. The same possibility is given for the subnet: you can select a valid subnet id for
+master and compute nodes or you can let ``pcluster configure`` set up everything for you. In that case,
+just select the configuration you prefer.
+
+.. code-block:: ini
+
+  Automate Subnet creation? (y/n) [y]: y
+  Allowed values for Network Configuration:
+  1. Master in a public subnet and compute fleet in a private subnet
+  2. Master and compute fleet in the same public subnet
+
+
+At the end of the process a message like this one will be shown:
+
+.. code-block:: ini
+
+  Configuration file written to /dir/conf_file
+  You can edit your configuration file or simply run 'pcluster create -c /dir/conf_file cluster-name' to create your cluster
+
 
 Now you can create your first cluster;
```

Comment:
This wasn't part of the changes introduced by this PR, but "cluster;" should be "cluster:"

Reply:
:+1: